apps/test-viewer/README.md

```diff
@@ -0,0 +1,63 @@
+# Test Viewer App
+
+This application provides test environment for developers working on widgets for ITwin applications.
+
+## Development
+
+Prior to running this app, you will need to [configure environment variables](#environment-variables).
+
+### Running app
+
+`npm start`
+
+### Running app with workspace packages hot reloading
+
+`npm run start:dev`
+
+By default `pnpm` symlinks dependencies to the modules directory. But this causes problems if workspace package `devDependencies` and app `dependencies` have different versions of the same package. To avoid this problem workspace packages should be [injected](https://pnpm.io/package_json#dependenciesmetainjected) into app. However, this breaks `webpack` hot reloading feature.
```

Comment:
I don't think it is possible to avoid this without forcing all pkgs to use same versions. It shouldn't be a problem when all pkgs used by test-viewer and test-viewer are updated to 4.0.0 but it will comeback as soon as one of those pkgs moves to new minor or patch versions. As that will require to update test-viewer deps and as a result other pkgs used in test-viewer.